development/build/scripts.js

```diff
@@ -122,11 +123,8 @@ function getPhishingWarningPageUrl({ config, testing }) {
     ? phishingWarningPageUrl
     : `${phishingWarningPageUrl}/`;
 
-  let phishingWarningPageUrlObject;
-  try {
-    // eslint-disable-next-line no-new
-    phishingWarningPageUrlObject = new URL(normalizedUrl);
-  } catch (error) {
+  const phishingWarningPageUrlObject = getURL(normalizedUrl);
+  if (!phishingWarningPageUrlObject) {
```

Comment:
`phishingWarningPageUrlObject` might be a string now. maybe
```suggestion
  const phishingWarningPageUrl = getURL(normalizedUrl);
  if (!phishingWarningPageUrl || phishingWarningPageUrl === '') {
```

Reply:
It looks like it's always either a URL object or an empty string, so the original code here should work (an empty string is falsy).

Seems unnecessarily confusing for `getURL` to work that way though.